consumer/consts.go

```diff
@@ -29,4 +33,8 @@ const (
 	clusterKey = "cluster"
 	// key for duration message type used in structured log messages
 	durationKey = "duration"
+	// key for data schema version message type used in structured log messages
+	versionKey = "version"
+	// currently supported data schema version
```

Comment:
Probably linters will freak out with this comment :-P

Reply:
They are, they are =)

Reply:
:rescue_worker_helmet: